internal/provider/resource_application.go

```diff
@@ -127,6 +127,11 @@ func resourceApplication() *schema.Resource {
 					},
 				},
 			},
+			"placement": {
+				Description: "Specify the target location for the application's units",
+				Type: schema.TypeString,
+				Optional: true,
+				Default: "",
```

Comment:
```suggestion
			"placement": {
				Description: "Specify the target location for the application's units",
				Type: schema.TypeString,
				Optional: true,
				Default: "",
			},
```

Merging main into this branch broke the code

Reply:
I'll take a look at what happened and submit fixes for what is preventing the build. This is what is happening when you try to build with the merged main code:
```
$ go build -o terraform-provider-juju
go: downloading github.com/rs/zerolog v1.29.0
go: downloading github.com/hashicorp/terraform-plugin-sdk/v2 v2.25.0
go: downloading github.com/hashicorp/go-hclog v1.4.0
go: downloading github.com/hashicorp/go-plugin v1.4.8
go: downloading github.com/hashicorp/terraform-plugin-go v0.14.3
go: downloading github.com/hashicorp/terraform-plugin-log v0.8.0
go: downloading golang.org/x/crypto v0.6.0
go: downloading golang.org/x/net v0.6.0
go: downloading google.golang.org/grpc v1.51.0
go: downloading golang.org/x/text v0.7.0
go: downloading golang.org/x/sys v0.5.0
go: downloading github.com/juju/worker/v3 v3.1.0
go: downloading github.com/hashicorp/terraform-registry-address v0.1.0
go: downloading github.com/hashicorp/hcl/v2 v2.16.1
go: downloading golang.org/x/term v0.5.0
# github.com/juju/terraform-provider-juju/internal/provider
internal/provider/resource_application.go:141:3: syntax error: unexpected newline in composite literal; possibly missing comma or }
internal/provider/resource_application.go:144:6: syntax error: unexpected resourceApplicationCreate, expecting (
internal/provider/resource_application.go:168:2: syntax error: non-declaration statement outside function body
internal/provider/resource_application.go:184:2: syntax error: non-declaration statement outside function body
```